src/maintenance/update_last_usage.py

```diff
@@ -0,0 +1,58 @@
+import os
+from datetime import datetime as dtime
+
+import delphi.operations.secrets as secrets
+import mysql.connector
+import redis
+
+REDIS_HOST = os.environ.get("REDIS_HOST", "delphi_redis")
+REDIS_PASSWORD = os.environ.get("REDIS_PASSWORD", "1234")
+LAST_USED_KEY_PATTERN = "*LAST_USED*"
+
+
+def redis_get_last_time_used(redis_cli, pattern: str):
+    keys = redis_cli.keys(pattern=pattern)
+    key_val_pairs = {
+        str(key).split("/")[1]: dtime.strptime(str(redis_cli.get(key)), "%Y-%m-%d").date() for key in keys
```

Comment:
just an idea, i dont know if this is actually any clearer or not...  feel free to ignore.
```suggestion
        f"{key}".split("/")[1]: dtime.strptime(f"{redis_cli.get(key)}", "%Y-%m-%d").date() for key in keys
```